spec/features/users/discover_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe "Discover Movies Page" do
+  before :each do
+    @user = create(:user)
+
+    visit user_discover_path(@user)
+  end
+
```

Comment:
I think we need to delete this file from your old tests

Reply:
Great catch